README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+Hey there!  My name is Kristina Wald.  Here are some quick facts about me:
+
+# Age
+25
+
+# Where I'm from
+Hyde Park - *grew up practically right on campus!*
+
+# What I'm studying
+Behavioral Science at Booth
+
+# Research interests
+* Person perception
+* Impression formation
+* Social interaction
+
+# Favorite things to do
+* Take walks *(especially along Lake Michigan!)*
+* Watch _**The Bachelor**_
```

Comment:
I'm kind of annoyed with Peter as the next bachelor. I like him, his mom seems just the right amount of crazy, but Mike would have injected some variety and fun.

Or a Property Brother. That would also be intriguing.

![](https://media.giphy.com/media/3JUr3oR5pD9ucqRCZr/giphy-downsized.gif)